landingpage/src/components/HeaderWithSearch/index.tsx

```diff
@@ -109,7 +112,12 @@ function HeaderWithSearch() {
 
 								<MyHr />
 
+								<TransactionTool title='Claim & Add inscriptions' mode='claimAndInscribe' name={queryString} />
```

Comment:
What do you think of expanding the Claim & Inscribe tool by default or just have it without dropdown?
Also what do you think about just "Claim & Inscribe"?

Reply:
Yes, it should be directly visible. I will remove the dropdown entirely.
"Claim & Inscribe" 💯